fastlane/Fastfile

```diff
@@ -155,73 +174,82 @@ in xcode user defined build configuration and just reflected in `Info.plist` dur
-        key: 'CFBundleVersion'
+    begin
+      desc "Build"
+      build_app(
+        app_identifier: bundle_identifier,
+        configuration: build_configurations[build_configuration],
+        match_type: match_types[build_configuration],
+        project_name: ENV["PROJECT_NAME"]
       )
-    )
 
-    desc "Bundle identifier from xCode project"
-    bundle_identifier = read_project_property(
-      build_configuration: build_configurations[:testflight],
-      build_setting: 'PRODUCT_BUNDLE_IDENTIFIER'
-    )
+      desc "Get rollbar server access token from configuration file."
+      rollbar_server_access_token = read_xcconfig_property(
+        xcconfig_path: project_xcconfig_path % [ENV['PROJECT_NAME'], build_configurations[build_configuration]],
+        xcconfig_key: 'ROLLBAR_SERVER_ACCESS_TOKEN'
+      )
 
-    desc "Build"
-    build_app(
-      app_identifier: bundle_identifier,
-      configuration: build_configurations[:testflight],
-      match_type: match_types[:testflight],
-      project_name: ENV["PROJECT_NAME"]
-    )
+      desc "In case rollbar server access token is present then do the uploading"
+      unless rollbar_server_access_token.to_s.empty?
 
-    desc "Get rollbar server access token from configuration file."
-    rollbar_server_access_token = read_xcconfig_property(
-      xcconfig_path: project_xcconfig_path % [ENV['PROJECT_NAME'], build_configurations[:testflight]],
-      xcconfig_key: 'ROLLBAR_SERVER_ACCESS_TOKEN'
-    )
+        desc "Upload dsym to rollbar."
+        upload_dsym(
+          access_token: rollbar_server_access_token,
+          version: String(next_build_number),
+          bundle_identifier: bundle_identifier,
+          dsym_zip_path: dsym_zip_path % ENV['PROJECT_NAME']
+        )
 
-    desc "In case rollbar server access token is present then do the uploading"
-    unless rollbar_server_access_token.to_s.empty?
+      end
 
-      desc "Upload dsym to rollbar."
-      upload_dsym(
-        access_token: rollbar_server_access_token,
-        version: String(next_build_number),
-        bundle_identifier: bundle_identifier,
-        dsym_zip_path: dsym_zip_path % ENV['PROJECT_NAME']
+      desc "Put back the backed up version number in `Info.plist`."
+      set_info_plist_value(
+        path: project_plist_path % ENV['PROJECT_NAME'],
+        key: 'CFBundleShortVersionString',
+        value: info_plist_backup_version_number
       )
 
+      desc "Put back the backed up build number in `Info.plist`."
+      set_info_plist_value(
+        path: project_plist_path % ENV['PROJECT_NAME'],
+        key: 'CFBundleVersion',
+        value: info_plist_backup_build_number
+      )
+
+      desc "Upload the built app to TestFlight."
+      publish_testflight
+
+    rescue => ex
+      print("Found the following exception: '#{ex}', resetting version changes")
+      reset_version_changes build_configuration: build_configuration, version: current_version_version
```

Comment:
I'd write this as:

```
reset_version_changes(
  build_configuration: build_configuration, 
  version: current_version_version
)
```

To keep the same style

Reply:
yeah, let's keep the same style. That said, this file doesn't follow the ruby department's standards (for instance, it's always preferred to avoid using parenthesis)